pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -733,7 +733,7 @@ stdenvNoCC.mkDerivation {
 
     # for substitution in utils.bash
     # TODO(@sternenseemann): invent something cleaner than passing in "" in case of absence
-    expandResponseParams = "${expand-response-params}/bin/expand-response-params";
+    expandResponseParams = lib.optionalString (expand-response-params != "") (lib.getExe expand-response-params);
```

Comment:
The change is okay because `-x` (executable) in  https://github.com/NixOS/nixpkgs/blob/01f064c99c792715054dc7a70e4c1626dbbec0c3/pkgs/build-support/wrapper-common/utils.bash#L128 
fails if `""`
```
$ test -x ""
$ echo $?
1
```